src/solvers/Solver.h

```diff
@@ -150,13 +148,13 @@ class LiveNurse : public Nurse {
 	// Data of the the particular period the live nurse is going to work
 	//----------------------------------------------------------------------------
 	int nbDays_, firstDay_;
+	const int originalNurseId_;
```

Comment:
We need the original nurse Id, but do we really need the live nurse to have an Id distinct from that of the original nurse?

Reply:
Yes, as the nurse id is used to get the position in the vector of live nurse. So the index could change especially with divideInConnexeComponent ... Perhaps we can change the name to index instead of id.